hutch_python/epics_arch.py

```diff
@@ -37,10 +47,21 @@ def main():
     parser = _create_parser()
     parsed_args = parser.parse_args()
     kwargs = vars(parsed_args)
+    logger_setup(parsed_args)
+    logger.debug("\nepicsarch-qs test script, git")
     create_arch_file(**kwargs)
 
 
-def create_arch_file(experiment, hutch=None, path=None, dry_run=False):
+def logger_setup(args):
+    # Setting up the logger, to show the level when enabled
+    logging.getLogger().addHandler(logging.NullHandler())
+    shown_logger = logging.getLogger('epicsarch-qs')
+    logger.setLevel(args.level)
+    logger.debug("Logger Level: ", logger.getEffectiveLevel())
+    logger.debug("Set logging level of %r to %r", shown_logger.name, args.level)
```

Comment:
This logger setup is a bit confusing to me.
`shown_logger` is created and then not used or modified. Then, we set up `logger`. Then we use `logger` to falsely report that we've set the log level of `shown_logger`.